ios/RCTWebRTC/WebRTCModule.m

```diff
@@ -50,7 +50,11 @@ - (instancetype)init {
         if (fieldTrials == nil) {
             // Fix for dual-sim connectivity:
             // https://bugs.chromium.org/p/webrtc/issues/detail?id=10966
-            fieldTrials = @{kRTCFieldTrialUseNWPathMonitor : kRTCFieldTrialEnabledValue};
+            fieldTrials = @{kRTCFieldTrialUseNWPathMonitor : kRTCFieldTrialEnabledValue,
+                // Initialize field trial for solving audio issues after hold when using CallKit.
+                // See: https://bugs.chromium.org/p/webrtc/issues/detail?id=8126#c35
+                @"WebRTC-Audio-iOS-Holding" : kRTCFieldTrialEnabledValue
```

Comment:
Please undo this, it's unrelated and known to cause crashes.

Reply:
Ooh, sorry i was making the patch for my use, forgot to switch the branch, it gets pushed in to this, I'll revert this, and as soon as a i get time, i will try completing this PR

Reply:
Sounds good!